docs/reference/reference-training/experiment-config-reference.rst

```diff
@@ -1268,32 +1415,32 @@ of records).
 
 .. note::
 
-   If the amount of data to train a model on is specified using records or epochs and the batch size
-   does not divide evenly into the configured number of inputs, the remaining "partial batch" of
+   When the amount of training data for a model is specified using records or epochs, and the batch
```

Comment:
Note, the whole Training Units section explains a few different config fields, but has now config fields all to itself.

So it's weird that it's sandwiched between actual config fields in the hierarchy (Profiling and Slurm Options).

Reply:
agree. resolved by moving the Training Units section to the top.